build/configs/rtl8721csm/security_hal_test/defconfig

```diff
@@ -41,39 +42,43 @@ CONFIG_RAW_BINARY=y
 # CONFIG_ARCH_FLOAT_H is not set
 # CONFIG_ARCH_STDARG_H is not set
 CONFIG_ARCH_HAVE_CUSTOMOPT=y
-CONFIG_DEBUG_NOOPT=y
+# CONFIG_DEBUG_NOOPT is not set
 # CONFIG_DEBUG_CUSTOMOPT is not set
-# CONFIG_DEBUG_FULLOPT is not set
+CONFIG_DEBUG_FULLOPT=y
 
 #
 # Chip Selection
 #
 CONFIG_ARCH_ARM=y
 # CONFIG_ARCH_XTENSA is not set
 CONFIG_ARCH="arm"
+# CONFIG_ARCH_CHIP_LM is not set
+# CONFIG_ARCH_CHIP_S5J is not set
+# CONFIG_ARCH_CHIP_BCM4390X is not set
+# CONFIG_ARCH_CHIP_STM32 is not set
+# CONFIG_ARCH_CHIP_IMXRT is not set
+# CONFIG_ARCH_CHIP_STM32L4 is not set
 CONFIG_ARCH_CHIP_AMEBAD=y
 CONFIG_ARCH_CHIP="amebad"
-CONFIG_RTL8721D=y
-# CONFIG_FPGA is not set
-CONFIG_CHIP_A_CUT=y
-# CONFIG_CHIP_B_CUT is not set
 
 #
 # ARM Options
 #
 # CONFIG_ARCH_CORTEXM3 is not set
 # CONFIG_ARCH_CORTEXM4 is not set
+# CONFIG_ARCH_CORTEXM7 is not set
 CONFIG_ARCH_CORTEXM33=y
+# CONFIG_ARCH_CORTEXR4 is not set
 CONFIG_ARCH_FAMILY="armv8-m"
 CONFIG_ARCH_HAVE_CMNVECTOR=y
 CONFIG_ARM_CMNVECTOR=y
 # CONFIG_ARCH_HAVE_FPU is not set
 # CONFIG_ARCH_HAVE_DPFPU is not set
-# CONFIG_ARCH_FPU is not set
-# CONFIG_ARM_HAVE_MPU is not set
-# CONFIG_ARM_MPU is not set
-# CONFIG_ARMV8M_MPU is not set
-# CONFIG_ARMV8M_MPU_NREGIONS is not set
+# CONFIG_ARM_HAVE_MPU_UNIFIED is not set
+CONFIG_ARM_MPU=y
+# CONFIG_ARMV7M_MPU is not set
+CONFIG_ARMV8M_MPU=y
```

Comment:
May I know why the security_hal_test config values are changed in this PR?
Is this config tested after these changes...Because mpu is not configured for secure context yet

Reply:
As discussed, removing changes from  security_hal_test config